ADOPTERS.md

```diff
@@ -0,0 +1,32 @@
+# Adopters
+
+This list captures the set of organizations that are using Crossplane within their environments. If
+you are an adopter of Crossplane and not yet on this list, we encourage you to add your organization
+here as well!
+
+The goal for this list is to be the complete and authoritative source for the entire community of
+Crossplane adopters, and give inspiration to others that are earlier in their Crossplane journey.
+
+Contributing to this list is a small effort that has a **big impact** to the projects growth,
```

Comment:
nitpick:
```suggestion
Contributing to this list is a small effort that has a **big impact** to the project’s growth,
```

Reply:
ah thanks for those eyes! 👀 make this correction